mod_ci/controllers.py

```diff
@@ -925,6 +928,66 @@ def progress_reporter(test_id, token):
     return "FAIL"
 
 
+def comment_pr(test_id, crash, pr_nr, platform):
+    """
+    Upload the test report to the github PR as comment
+
+    :param test_id: The identity of Test whose report will be uploaded
+    :type test_id: str
+    :crash: whether test results in crash or not
+    :type: boolean
+    :pr_nr: PR number to which test commit is related and comment will be uploaded
+    :type: str
+    """
+    from run import app, log
+    regression_testid_passed = g.db.query(TestResult.regression_test_id).filter(
+        and_(TestResult.test_id == test_id,
```

Comment:
All these and_'s can be removed according to the documentation: http://docs.sqlalchemy.org/en/latest/orm/query.html#sqlalchemy.orm.query.Query.filter

Just using a list of all of them should be working, and should make this a bunch more readable.